tasks/main.yml

```diff
@@ -200,10 +200,12 @@
     name: glibc-common
     state: latest
   become: yes
+  when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
 
 - name: reinstall glibc-common to lay down langs
   become: true
   command: yum -y reinstall glibc-common
+  when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
```

Comment:
Same as above:
```suggestion
  when: ansible_distribution != 'Debian'
```

Reply:
... and here it is additionally using 'yum'.
Can we test for 'redhat-family'?

Reply:
In this case I would accept the original solution. Thanks!